routes/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (express) {
   router.use('/user', require('./users')(express))
   router.use('/post', require('./posts')(express))
   router.use('/feed', require('./feed')(express))
+  router.use('/search', require('./search')(express))
```

Comment:
I honestly think this depends on if we have different use cases for each of the endpoints.